### ISSUE TITLE
Test failing on Mac

### ISSUE THREAD

**Issue by dfm** [ColCarroll/minimc#5]
On my machine, the following tests fail on the master branch:

```
test_hamiltonian_monte_carlo[leapfrog_twostage]
test_hamiltonian_monte_carlo_mv
```

even though they pass on Travis.

**Comment by ColCarroll** [comment]
Yeah, I am guessing this is because of the RNGs, but haven't dug in. I keep loosening the tolerances on these tests. Feel free to loosen some more if it improves the development experience.

**Comment by eigenfoo** [comment]
This is probably closed by #7? At least, those two tests passed on my machine, which is a MacBook Pro 🤷‍♂
